src/get-started/java-binding.md

```diff
@@ -242,6 +242,35 @@ synchronous I/O in this handler, as it is invoked synchronously in the same
 thread that handles transactions. Blocking that thread will delay transaction
 processing on the node.
 
+### Core Schema API
+
+Users can access blockchain state using methods of [`Blockchain`][blockchain]
```

Comment:
As it is not _immediately_ clear where this might be used, I'd clarify that 
(1) not just "blockchain state", but information stored in the blockchain by the framework (i.e., it is _not_ used to access the state of services, but some meta-information about the system)
(2) this API can be used both in transaction code and in read requests.